rachelekm/vegishphilly
# Add HomePage and display restaurants

## Overview

This creates a public homepage at route "/" that displays restaurants within the bounds of the Map bounding box. Restaurants are plotted on the Map and their information is displayed in the Sidebar. On Map move, restaurants and their info should synchronously update according to the new bounding box and similarly update when a user interacts with pagination in the Sidebar: "next", "previous", or selecting page number. 
Connects #4 

### Demo

HomePage with paginated restaurants displayed in Map and Sidebar:
![Screen Shot 2021-09-07 at 1 58 26 PM](https://user-images.githubusercontent.com/36374797/132391744-0a138642-2f68-4fd2-9531-c05cbe149cf7.png)
On selecting "next" or page 2, Map and Sidebar update restaurants:
![Screen Shot 2021-09-07 at 1 58 41 PM](https://user-images.githubusercontent.com/36374797/132391835-effeb2f7-f303-4f49-9895-00086e302298.png)


### Notes

I deleted frontend auth functionality (including: fetch to get the current User info, PrivateRoute, route to "/auth") since it doesn't fit current needs after making HomePage public and had the potential to create ts unused variable errors. Will eventually have to add this back in for authorizing users on sign in and to leave ratings.

## Testing Instructions

 * Start server and open http://localhost:4567 in browser
 * Confirm HomePage displays on route "/"
 * Confirm that restaurants are paginated and display on both Map and Sidebar
 * Confirm that paginated restaurants update on Map move as well as on "next", "previous", and when selecting page number in Sidebar
